### ISSUE TITLE
Web - Steps Page - User Can Not Tell Where They are on the Screen Visually

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#109]
Users have no visual indication of where they are on in the screen when tabbing through focus-able items. Also, for some reason when I tab on the Steps pages and others my screen reader JAWS starts reading from the top of the page instead of the next items in the tab order — going to open another issue for this problem because it's confusing.

**Comment by KyleLivingstonDOL** [comment]
Thanks John. I think we can close this issue since it is addressed on issue #110 . Correct?

**Comment by jfc3-dol** [comment]
@KyleLivingstonDOL,

These are two different issues, were the one you can't tell when tabbing on the page where the user is, and the other is where the screen reader focuses.

**Comment by jfc3-dol** [comment]
Not able to see in IE 11 when tabbing through the main Step page when I am in the main content section of the page. Please use "Step 1: Engage Stakeholders and Partners" ( https://dolcontentdev.opadev.dol.gov/ilab/complychain/steps/3 ) as an example.

It works in Firefox or Chrome and not even displaying the page in Edge.

**Comment by Abychalil** [comment]
We have an issue like, the focus is going inside the Menu and reads the menu content even if the menu is shown closed. so the focus rings will be visible on the right side and will come to the  "why develop" link only after reading all the menu contents. This happens especially when we open the app for the first time or in a new window.  Do you think for this reason that the user cannot say where they are on the screen visually? 

Or the issue is that the focus rings are not visible on the web elements?

@jfc3-dol

**Comment by jfc3-dol** [comment]
Yes, in IE 11, it's tabbing through the Menu items when it's closed, and it should be doing that.

It's also not showing me where I am between the Menu button and the Previous and Next page links/buttons. I would expect to see where I am at when I'm for the page Bookmark or any of the show/hide content section buttons too.

Please see the attached screenshot.

![ilab - comply chain - step 1 tabbing issues in ie 11](https://user-images.githubusercontent.com/10423487/87081155-d6517680-c1f6-11ea-9e46-13fbda3dbee4.png)

**Comment by Abychalil** [comment]
@jfc3-dol 
I can view the focus rings moving on all the accordions while tabbing in JAWS. I am wondering why these changes not reflecting in your system! I am attaching a screenshot for yout to refer. Hope you are provided with updated version or can you please try after clearing the cache.

![Focus ring](https://user-images.githubusercontent.com/49654920/87091436-a9f22600-c207-11ea-8fea-eb86108ace78.PNG)
our reference. [](url)

**Comment by Abychalil** [comment]
@jfc3-dol 
![jaws_focus rings](https://user-images.githubusercontent.com/49654920/87091687-19681580-c208-11ea-9dda-b40e63420cf3.PNG)

I can view the focus rings moving on all the accordions while tabbing in JAWS. I am wondering why these changes not reflecting in your system! I am attaching a screenshot for yout to refer. Hope you are provided with updated version or can you please try after clearing the cache.

**Comment by Abychalil** [comment]
@jfc3-dol 
I am not sure why its showing different in your system.  1 or 2 other issues  are also working in my system, issue like 'Menu stays open even after focus moves away'. works well in my system. I just discussed with the developer  about this and he is also saying the same, its working fine for him too. He just asked to clear the cache and try again.

**Comment by jfc3-dol** [comment]
@Abychalil 

I wasn't using JAWS when doing the tabbing at this point. I was tabbing, along with no assistive technology. I even started with a new browser, meaning It was closed, and I re-opened it and still get the same issue in IE 11.

I even checked with JAWS and didn't see the focus ring or outline for the items mentioned above when tabbing to the items or even reading them.

**Comment by jfc3-dol** [comment]
@Abychalil 

When tabbing without a screen reader or assistive technology I only see a focus ring in Chrome for this or the Step URLs.

https://dolcontentdev.opadev.dol.gov/ilab/complychain/steps/1

Nothing show on the show hide content in IE 11, Edge, or Firefox.

**Comment by jfc3-dol** [comment]
@harpreetsdev @Abychalil @pbhatt17

I am closing this issue because I can see the focus ring/border for IE 11, Edge, Firefox, and Chrome now.

Going to open another issue for the part where on the Steps page, even when the Menu is closed, the user has to tab through those items in most of the browsers.
